website/pages/docs/customization/config-functions.md

```diff
@@ -0,0 +1,339 @@
+  transform(props) {
+    return {
+      srOnly: true,
+      ...props
+    }
+  }
+})
+```
+
+### `definePreset`
+
+Function for [preset](/docs/customization/presets#creating-a-preset) definitions.
+
+```ts
+import { definePreset } from '@pandacss/dev'
+
+export const pandaPreset = definePreset({
+  theme: {
+    extend: {
+      tokens: {
+        colors: { primary: { value: 'blue.500' } }
+      }
+    }
+  }
+})
+```
+
+### `defineKeyframes`
+
+Function for [keyframes](/docs/customization/theme#keyframes) definitions.
+
+```ts
+import { defineKeyframes } from '@pandacss/dev'
+
+export const fadein = defineKeyframes({
+  '0%': { opacity: '0' },
+  '100%': { opacity: '1' }
+})
+```
+
+### `defineGlobalStyles`
+
+Function for [global styles](/docs/concepts/writing-styles#global-styles) definitions.
+
+```ts
+import { defineGlobalStyles } from '@pandacss/dev'
+
+const globalCss = defineGlobalStyles({
+  'html, body': {
+    color: 'gray.900',
+    lineHeight: '1.5'
+  }
+})
+```
+
+### `defineUtility`
+
+Function for [utility](/docs/customization/utilities) definitions.
+
+```ts
+import { defineUtility } from '@pandacss/dev'
+
+export const br = defineUtility({
+  className: 'rounded',
+  values: 'radii',
+  transform(value) {
+    return { borderRadius: value }
+  }
+})
+```
+
+### `defineTextStyles`
+
+Function for [text styles](/docs/theming/text-styles) definitions.
+
+```ts
+import { defineTextStyles } from '@pandacss/dev'
+
+export const textStyles = defineTextStyles({
+  body: {
+    description: 'The body text style - used in paragraphs',
+    value: {
+      fontFamily: 'Inter',
+      fontWeight: '500',
+      fontSize: '16px',
+      lineHeight: '24',
+      letterSpacing: '0',
+      textDecoration: 'None',
+      textTransform: 'None'
+    }
+  }
+})
+```
+
+### `defineLayerStyles`
+
+Function for [layer styles](/docs/theming/layer-styles) definitions.
+
+```ts
+import { defineLayerStyles } from '@pandacss/dev'
+
+const layerStyles = defineLayerStyles({
+  container: {
+    description: 'container styles',
+    value: {
+      bg: 'gray.50',
+      border: '2px solid',
+      borderColor: 'gray.500'
+    }
+  }
+})
+```
+
+### `defineStyles`
+
+Function for [style](/) definitions.
+
+This comes in handy when you want to define reusable styles in the config.
+
+E.g. a set of styles to be used in multiple variants within a [recipe](/docs/concepts/recipes#config-recipe).
+
+```ts {3, 14, 18} filename="recipes/button.ts"
+import { defineRecipe, defineStyles } from '@pandacss/dev'
+
+const buttonVisualStyles = defineStyles({
+  borderRadius: 'lg',
+  boxShadow: 'sm'
+})
+
+export const buttonRecipe = defineRecipe({
+  // ...
+  variants: {
+    visual: {
+      funky: {
+        bg: 'red.200',
+        color: 'white',
+        ...buttonVisualStyles
+      },
+      edgy: {
+        border: '1px solid {colors.red.500}',
+        ...buttonVisualStyles
+      }
+    }
+  }
+})
+```
+
+## Token Creators
```

Comment:
In the token docs, please add a link to this section of the docs so it's easier to find.